### ISSUE TITLE
incompatibility with python 3.12

### ISSUE THREAD

**Issue by slynobody** [evilsocket/opensnitch#1132]
**Describe the bug**
does not work anymore with python 3.12

Include the following information:
 - OpenSnitch 1.6.5.1
 - OS: Debian
 - Version: trixie
 - Window Manager: KDE
 - Kernel version: 6.7

**To Reproduce**
Describe in detail as much as you can what happened.

1. work with os with python 3.11, everything works
2. install python3.12
3. opensnitch-ui throws errors

**Post error logs:** 
  File "/usr/bin/opensnitch-ui", line 41, in <module>
    from opensnitch.service import UIService
  File "/usr/lib/python3/dist-packages/opensnitch/service.py", line 18, in <module>
    from opensnitch.dialogs.prompt import PromptDialog
  File "/usr/lib/python3/dist-packages/opensnitch/dialogs/prompt.py", line 17, in <module>
    from opensnitch.desktop_parser import LinuxDesktopParser
  File "/usr/lib/python3/dist-packages/opensnitch/desktop_parser.py", line 3, in <module>
    import pyinotify
  File "/usr/lib/python3/dist-packages/pyinotify.py", line 71, in <module>
    import asyncore
ModuleNotFoundError: No module named 'asyncore'

**Comment by slynobody** [comment]
is there a workaround? python3.12 is on sid now replacing 3.11?

**Comment by gustavo-iniguez-goya** [comment]
I've just released  v1.6.6 with a temporal solution. It should not crash now.

**Comment by gustavo-iniguez-goya** [comment]
Fedora "fixed" this issue almost  a year ago:

https://src.fedoraproject.org/rpms/python-inotify/c/68a5a14134b8cefe47f1789dd444056abd8a70b8?branch=rawhide

**Comment by gustavo-iniguez-goya** [comment]
And now a fix has entered in Debian Sid :tada: 

```
  * Add pyasyncore to Depends for support in Python >= 3.12.
    (Closes: #1075811, #1040102, #1075939)
```
